test/test_autograd.py

```diff
@@ -2952,6 +2952,12 @@ def test_pow_scalar_base(self):
         a = torch.arange(1, 13, dtype=torch.double).view(3, 4).requires_grad_()
         gradcheck(lambda a: torch.pow(2, a), (a,))
 
+    def test_sinc(self):
+        # The derivative of sinc(x) at x=0 has to be special cased.
+        # A naive computation will result in 0/0 -> NaN.
+        a = torch.Tensor([0.0, 1.0], dtype=torch.double).requires_grad_()
```

Comment:
nit:
- Use `torch.tensor()` (lower t) as the constructor. The one with capital T is the old one.
- You can pass `requires_grad=True` as a keyword argument directly.

Reply:
Done